pkg/controllers/management/secretmigrator/authconfig.go

```diff
@@ -116,6 +117,46 @@ func (h *handler) migrateShibbolethSecrets(unstructuredConfig map[string]any) (*
 	return shibbConfig, nil
 }
 
+// migrateOKTASecrets effects the migration of secrets for the OKTA provider.
```

Comment:
affects instead of effects

Reply:
He might have meant `perform` (`to effect something` ~~ `to perform/do something`)

Reply:
Yeah, it's _effects_ as in _cause (something) to happen; bring about._